Performance-Tracing.md

```diff
@@ -67,12 +67,12 @@ The goal of this process is to be able to extract a reduced repro for which you
     5.  This mode is for zooming.
         Click-drag up or right to zoom in and down or left to zoom out.
 
-    *Aside:* the [edge://tracing](edge://tracing) tracing controls are rather clumsy.
-    An alternative is to open the trace in the F12 tools by clicking the Upload button on the Performance tab.
-    This has much more modern navigation, but drops instant events (mostly used for errors in compiler traces) and possibly other features.  We hope to improve support in future releases.
+    *Aside:* the [`about://tracing`](about://tracing) UI is rather clumsy.
+    An alternative is to open the trace in Devtools (F12) by clicking the Upload button on the Performance tab.
+    This provides a more navigation experience, but drops instant events (mostly used for errors in compiler traces) and possibly other features.
```

Comment:
Missing a word?  "a more navigation experience"

Reply:
Yeah, I don't know how I dropped it in the copy+paste from daniel's suggestion...